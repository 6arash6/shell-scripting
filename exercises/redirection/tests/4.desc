this should check the output of last-5-records.sh
