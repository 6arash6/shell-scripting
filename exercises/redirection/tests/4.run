./first-engineer.sh ./data/yellow-pages.txt
