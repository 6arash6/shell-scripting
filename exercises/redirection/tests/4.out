engineer
Quinn Anderson
7556 Elm St.
Copenhagen, Denmark
